src/pages/item/item-actions/ItemActions.tsx

```diff
@@ -21,29 +31,98 @@ export default function ItemActions(props: { itemId: string | number }) {
 				"Can't load Downstream References - data still loading or failed to do so."
 			);
 		}
+		
+	};
+
+	//get all downstreamRefs and associations and then call in the miro.api a method which creates all connectors
+	// { downstreamRefs: [number], associations: [{associationId: number, targetItemId: number}] }
+	React.useEffect(() => {
+		async function fetchData() {
+			if (data && (data.downstreamReferences.length || data.outgoingAssociations.length)) {
+			var downstreamRefs: React.SetStateAction<number[]> = [];
+			var associations: ((prevState: never[]) => never[]) | { associationId: number; targetItemId: number; }[] = [];
+			if (data.downstreamReferences.length) {
+				data.downstreamReferences.forEach(function (downstreamReference) {
+				downstreamRefs.push(downstreamReference.itemRevision.id);
+				});
+			}
+			if (data.outgoingAssociations.length) {
+				data.outgoingAssociations.forEach(function (outgoingAssociation) {
+				var association = {associationId: outgoingAssociation.id, targetItemId: outgoingAssociation.itemRevision.id};
+				associations.push(association);
+				});
+			}
+
+			//check if downstreamReferences or outgoingAssociations exist on the board
+			const amountOfCardsOnBoard = await calculateAmountOfCardsOnBoard();
+
+			if(amountOfCardsOnBoard){
+				setShowDependenciesDisabled(false)
+			}
+			else{
+				setShowDependenciesDisabled(true)
+			}
```

Comment:
Just saying that this could neatly be written in a line as `setShowDependenciesDisabled(amountOfCardsOnBoard > 0)`

Reply:
You set the expression to `amountOfRelationsOnBoard < 0`, but the minimum value that `calculateAmountOfRelationsOnBoard` will return is 0, isn't it? So this will never be true?